src/main/java/frc/robot/RobotContainer.java

```diff
@@ -178,7 +179,11 @@ private void configureButtonBindings() {
             rollerSubsystem.setRollPower(rollPower);
         }, rollerSubsystem));
 
-        mechAButton.onTrue(new InstantCommand(rollerSubsystem::openMotor, rollerSubsystem));
+        if(driveSubsystem instanceof SwerveSubsystem){
```

Comment:
Does this need to be specifically a `SwerveSubsystem` and not just a `BaseSwerveSubsystem`? `SwerveSubsystem` doesn't define any additional methods, and you can just put this button binding in the `BaseSwerveSubsystem` if statement earlier in the method otherwise.